architecture/topics/sticky_sessions.adoc

```diff
@@ -22,15 +20,17 @@ Cluster administrators can turn off stickiness for passthrough routes separately
 from other connections, or turn off stickiness entirely.
 
 By default, sticky sessions for passthrough routes are implemented using the
-`source` xref:load-balancing[load balancing strategy]. The default can be
-changed for all passthrough routes by using the `ROUTER_TCP_BALANCE_SCHEME`
-xref:env-variables[environment variable], and for individual routes by using the
+`source` xref:#load-balancing[load balancing strategy]. However, the
+`round-robin` xref:#load-balancing[load-balancing strategy] is the default when
```

Comment:
Should be `roundrobin` instead of `round-robin`.

Reply:
@lihongan thanks!